bigbluebutton-html5/imports/ui/components/whiteboard/shapes/helpers.js

```diff
@@ -0,0 +1,51 @@
+// applies zooming to the stroke thickness
+const zoomStroke = (thickness, widthRatio, heightRatio) => {
+  let ratio;
+
+  ratio = (widthRatio  + heightRatio) / 2;
+  return thickness * 100 / ratio;
+};
+
+const formatColor = (color) => {
+  //let color = this.props.shape.shape.shape.color;
+  if (color == null) {
```

Comment:
Then you dont need to compare with null, just use `if(color)` then if its null or undefined will be false.